pkgs/top-level/python-packages.nix

```diff
@@ -1115,6 +1115,8 @@ in {
     inherit (pkgs) pkgconfig;
   });
 
+  pycoin = callPackage ../development/python-modules/pycoin { };
```

Comment:
This probably shouldn't be so high up in the order. I would expect this would go farther down, in the line number >2000 range. I think this range is loosely reserved for important/high-demand packages.